akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -18,8 +18,10 @@ import akka.event.Logging.simpleName
  */
 private[akka] object StreamLayout {
 
-  // compile-time constant
-  final val Debug = true
+  // Turn this on to add run-time checks about Stream consistency during
```

Comment:
it is about consistency of the underlying data structure describing stream layouts (Stream consistency would mean something more general)